### PR TITLE
Chore/improve test coverage

### PR DESCRIPTION
**Pull request by tsubik**
Little refactoring. Adding more specs, removing not needed files, vol1.

### CODE REVIEW

File: app/controllers/v1/observation_filters_controller.rb [hunk 1]
```diff
@@ -255,7 +255,7 @@ def valid_params(name, value)
     # TODO: Hack. Modify to use refinements
     def self.is_of_type(type, value)
       if type == Integer
-        return true if value.to_i.to_s == self
+        return true if value.to_i.to_s == value
```
Comment (santostiago) [inline app/controllers/v1/observation_filters_controller.rb:258]: 😮

File: app/models/concerns/globalize_fallback.rb [hunk 2]
```diff
@@ -1,43 +0,0 @@
-# frozen_string_literal: true
```
Comment (santostiago) [inline app/models/concerns/globalize_fallback.rb:1]: Mmmm... so how do we do now when there's a field missing in translations?

**Review by santostiago — APPROVED** [review]
I just have a question regarding the globalize fallback.

File: app/models/concerns/globalize_fallback.rb (hunk 2 above)
Comment (tsubik) [reply app/models/concerns/globalize_fallback.rb:1]: @santostiago that module is not used anywhere, at least I did not find any reference to it. Code coverage of this code was also 0%

**Pull request merged**